Tests/scripts/lock_cloud_machines.py

```diff
@@ -182,6 +206,18 @@ def get_my_place_in_the_queue(storage_client: storage.Client, gcs_locks_path: st
     previous_build_in_queue = ''
     if my_place_in_the_queue > 0:
         previous_build_in_queue = sorted_builds_in_queue[my_place_in_the_queue - 1].get('name')  # type: ignore[assignment]
+
+    try:
+        if my_place_in_the_queue != my_places[-1]:
```

Comment:
why do you need it to be a list, since you're only using the last item always? and in my next comment, you can see it has no value to pint it.